packages/web/docs/src/pages/docs/api-reference/cli.mdx

```diff
@@ -453,3 +453,15 @@ hive schema:check schema.graphql --github
 
 List of all available CLI commands and their options can be found
 [here](https://github.com/graphql-hive/platform/blob/main/packages/libraries/cli/README.md#commands)
+
```

Comment:
any recommendations for auto generating this content?

Reply:
I guess you could use `export async function getStaticProps() { }` API to render content based on the `packages/libraries/cli/src/helpers/errors.ts` in some way.

Reply:
Im moving the documentation portion to a separate PR and am looking into this. ty